portfolio/src/main/webapp/style.css

```diff
@@ -1,10 +1,153 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    height: 100%;
+    width: 100%
 }
+ 
+body,
+h1,
+h2,
+h3
+h4,
+h5,
+h6 {
+    font-weight: 300
+}
+ 
+body {
+    font-weight: 600
+}
+ 
+.struct {
+    font-size: 20px;
+    font-weight: 400
+}
+ 
+p {
+    font-weight: 400;
+    font-family: Avenir
+
+}
+
+
```

Comment:
nit: make sure to have a consistent spacing between the different items. I see 3 empty lines here vs. 1 for previous elements